api/handlers.go

```diff
@@ -742,6 +742,10 @@ func (api *API) userPUT(u *database.User, w http.ResponseWriter, req *http.Reque
 		changedEmail = true
 	}
 
+	if api.staticDeps.Disrupt("DependencyUserPutMongoDelay") {
```

Comment:
So my original suggestion included something like this, so we only sleep half the time, this would ensure other transactions that are submitted afterwards and don't sleep get executed first. 
```suggestion
	if api.staticDeps.Disrupt("DependencyUserPutMongoDelay") && fastrand.Int(0,2) == 0 {
```